src/priv-preface.adoc

```diff
@@ -28,7 +28,8 @@ _Supervisor ISA_ +
 *Sscofpmf* +
 _Ssdbltrp_ +
 *Hypervisor ISA* +
-_Shlcofideleg_
+_Shlcofideleg_ +
+_Svvptc_
```

Comment:
For future reference, note that you should format ratified specs as bold `*Svvptc*` rather than italicized `_Svvptc_`.  I'll fix this one up manually.

Reply:
Thanks. I missed that.